public/app/features/alerting/unified/components/Provisioning.tsx

```diff
@@ -10,13 +10,16 @@ export enum ProvisionedResource {
   RootNotificationPolicy = 'root notification policy',
 }
 
-interface ProvisioningAlertProps {
+// we'll omit the props we don't want consumers to overwrite and forward the others to the alert component
+type ExtraAlertProps = Omit<ComponentPropsWithoutRef<typeof Alert>, 'title' | 'severity'>;
+
+interface ProvisioningAlertProps extends ExtraAlertProps {
   resource: ProvisionedResource;
 }
 
-export const ProvisioningAlert = ({ resource }: ProvisioningAlertProps) => {
+export const ProvisioningAlert = ({ resource, ...rest }: ProvisioningAlertProps) => {
```

Comment:
Changes here are to allow passing `bottomSpacing={0}` because the Alert component applies margins by default :(